docs/learn/response_models.md

```diff
@@ -156,6 +170,33 @@ In the above example, your IDE will provide proper autocompletion and type check
 
 When using Response Models, the default setting is `json_mode=False`, which will use [`Tools`](./tools.md) under the hood to perform the structured output extraction. However, for some providers setting `json_mode=True` enables support for field types that may not be supported by tools.
 
+```python hl_lines="8"
+from mirascope.core import openai, prompt_template
+from pydantic import BaseModel
+
+class ComplexBook(BaseModel):
```

Comment:
what makes this "complex" vs. something more descriptive like `BookOutline` or `BookWithChapterTitles`?